bundles/binding/org.openhab.binding.isy/OSGI-INF/binding.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2010-2015, openHAB.org and others.
+
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
```

Comment:
Run `mvn license:format -Dtycho.mode=maven` in the bundles/binding/org.openhab.binding.isy directory (while on this PR branch) to update license headers.